### PR TITLE
docs: Added CONTRIBUTING guide and integrated into README

### PR DESCRIPTION
**Pull request by hferentschik**
- adding information about building from source
- fixing clean target in Makefile

fixes issue #56

### CODE REVIEW

**Comment by rawlingsj** [comment]
retriggering CI as prod infra nodes recycled

/test this

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contribution guidelines
+
+## Filing issues
+
+File issues using the standard [Github issue tracker](https://github.com/redhat-developer/openshift-vsts/issues) for this repository.
```
Comment (deanesmith) [inline CONTRIBUTING.md]: This - https://github.com/redhat-developer/openshift-vsts/issues

Comment (hferentschik) [reply CONTRIBUTING.md]: Bugger, let me fix this ;-)

Comment (hferentschik) [reply CONTRIBUTING.md]: I pushed an update.

**Comment by deanesmith** [comment]
Look good to merge.  This will also bump the build and hopefully fixes our missing version

**Review by deanesmith — APPROVED** [review]

**Pull request merged**